src/applications/_mock-form-ae-design-patterns/mocks/endpoints/user/index.js

```diff
@@ -0,0 +1,1533 @@
+const set = require('lodash/set');
+const cloneDeep = require('lodash/cloneDeep');
+/**
+ * Loops through the claims array and adds the claims to the user object
+ *
+ * @param {Object} user
+ * @param {Array} claims
+ * @returns user with claims added to the profile.claims object with boolean values
+ */
+const createUserWithDataClaims = (user, claims) => {
+  const result = cloneDeep(user);
+  claims.forEach(claim => {
+    result.data.attributes.profile.claims[claim.name] = claim.value;
+  });
+  return result;
+};
+
+const baseUserResponses = {
+  loa1User: {
+    data: {
+      id: '',
+      type: 'users_scaffolds',
+      attributes: {
+        services: [
+          'facilities',
+          'hca',
+          'edu-benefits',
+          'form-save-in-progress',
+          'form-prefill',
+          'evss-claims',
+          'form526',
+          'user-profile',
+          'appeals-status',
+          'id-card',
+          'identity-proofed',
+          'vet360',
+        ],
+        account: {
+          accountUuid: '7d9e2bfb-13ae-45c8-8764-ea3c87cd8af3',
+        },
+        profile: {
+          email: 'vets.gov.user+75@gmail.com',
+          firstName: 'MITCHELL',
+          middleName: 'G',
+          lastName: 'JENKINS',
+          birthDate: '1949-03-04',
+          gender: 'M',
+          zip: '97063',
+          lastSignedIn: '2022-03-24T18:15:06.566Z',
+          loa: {
+            current: 1,
+          },
+          multifactor: false,
+          verified: false,
+          signIn: {
+            serviceName: 'idme',
+            accountType: 'N/A',
+          },
+          authnContext: 'http://idmanagement.gov/ns/assurance/loa/3',
+          claims: {
+            ch33BankAccounts: false,
+            communicationPreferences: false,
+            connectedApps: false,
+            militaryHistory: false,
+            paymentHistory: false,
+            personalInformation: false,
+            ratingInfo: false,
+            appeals: false,
+            medicalCopays: false,
+          },
+        },
+        vaProfile: {
+          status: 'OK',
+          birthDate: '19490304',
+          familyName: 'Jenkins',
+          gender: 'M',
+          givenNames: ['Mitchell', 'G'],
+          isCernerPatient: false,
+          facilities: [
+            {
+              facilityId: '989',
+              isCerner: false,
+            },
+            {
+              facilityId: '987',
+              isCerner: false,
+            },
+            {
+              facilityId: '983',
+              isCerner: false,
+            },
+          ],
+          vaPatient: false,
+          mhvAccountState: 'NONE',
+        },
+        veteranStatus: {
+          status: 'OK',
+          isVeteran: true,
+          servedInMilitary: true,
+        },
+        inProgressForms: [],
+        prefillsAvailable: ['21-686C', 'mock-form-ae-design-patterns'],
```

Comment:
I think that because we changed the id to be 'FORM_MOCK_AE_DESIGN_PATTERNS' that is what should be present here instead of 'mock-form-ae-design-patterns' so that it show correctly. Did you test the prefill logic once we change the from id in the constants?

Reply:
done